docs/releases/v2020.2.1.rst

```diff
@@ -0,0 +1,47 @@
+Gluon 2020.2.1
+===============
+
+Added hardware support
+----------------------
+
+- add support for factory images for TP-Link CPE210 3.20 (`#2080 <https://github.com/freifunk-gluon/gluon/issues/2080>`_)
```

Comment:
- Start with an uppercase letter
- This also added the aliases needed for updates
- Let's not mix tenses in the release notes. I think we usually write our release notes in past tense ("Added support ...")

Reply:
done